### PR TITLE
Extractor update for 1.13

### PR DESCRIPTION
**Pull request by lluiscab**
I've updated the extractor so it correctly gets all required data.
https://github.com/Pokechu22/Burger/tree/1.13-fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # Burger-extractor
 
 Convert [burger](https://github.com/mcdevs/burger) json to
-[minecraft-data](https://github.com/PrismarineJS/minecraft-data) json files : items, blocks, biomes and recipes.
+[minecraft-data](https://github.com/PrismarineJS/minecraft-data) json files.
```
Comment (rom1504) [inline README.md]: Please keep the information on what files it generates. It doesn't generate everything in Minecraft data

**Comment by rom1504** [comment]
Can you add standard (npm install --saveDev standard and add standard in the test script in package.json) and run standard --fix ?

**Comment by lluiscab** [comment]
yep, didn't really have this here to my IDE config just took over the standard. Will do tomorrow

**Comment by lluiscab** [comment]
Just standardized the code.

**Review by rom1504 — CHANGES_REQUESTED** [review]
looks ok but let's wait for the mcdata PR to be done before merging this as it might require changes here too

File: README.md (hunk 1 above)
Comment (lluiscab) [reply README.md]: Okay

**Comment by lluiscab** [comment]
Okay, just wanted to get the PR started here.

**Pull request merged**